src/linux/mod.rs

```diff
@@ -88,8 +87,18 @@ impl<'a> Iterator for SegmentIter<'a> {
     }
 }
 
+impl<'a> fmt::Debug for SegmentIter<'a> {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        let ref phdr = self.inner.as_slice()[0];
+
+        write!(f, "SegmentIter {{ phdr: ")?;
+        fmt_debug_phdr(phdr, f)?;
+        write!(f, " }}")
```

Comment:
Nitpick: This should be 

```rust
f.debug_struct().field("phdr", &DebugPhdr(phd)).finish()
```

Reply:
Cool, I guess I learn something new everyday.